Formula/muparser.rb

```diff
@@ -15,13 +15,12 @@ class Muparser < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "gcc"
-
-  fails_with :clang # no OpenMP support
 
   def install
     mkdir "build" do
-      system "cmake", "..", *std_cmake_args
+      cmake_args = std_cmake_args
+      cmake_args << "-DENABLE_OPENMP=OFF"
+      system "cmake", "..", *cmake_args
```

Comment:
This is a little cleaner
```suggestion
      system "cmake", "..", *std_cmake_args, "-DENABLE_OPENMP=OFF"
```